core/core.go

```diff
@@ -63,7 +63,7 @@ func (c *Core) InsertChain(blocks types.Blocks) (int, error) {
 		// if the order of the block is less than the context
 		// add the rest of the blocks in the queue to the future blocks.
 		if !isCoincident && !domWait {
-			_, newPendingEtxs, err := c.sl.Append(block.Header(), common.Hash{}, big.NewInt(0), false, true, block.ManifestHash())
+			_, newPendingEtxs, err := c.sl.Append(block.Header(), common.Hash{}, big.NewInt(0), false, true, block.ManifestHash(), nil)
```

Comment:
NewInboundEtxs empty here ? because its not part of the body ?

Reply:
No, `newInboundEtxs` is the list of new ETXs passed in from our dom. Since `InsertChain` is not invoked from the dom, then this list is empty. I should probably rename the arg to `newInboundEtxsFromDom` or something

Reply:
Yes, i checked. Looks like every in context Append call is passed with `nil` for `NewInboundEtxsFromDom`.